src/store/DicomStoreSDW.js

```diff
@@ -0,0 +1,144 @@
+import fs from "fs/promises";
+import crypto from "crypto";
+import { saveJson } from "../utils/index.js";
+import { DicomStoreAccess } from "./DicomStoreAccess.js";
+
+export class DicomStoreSDW extends DicomStoreAccess {
```

Comment:
The store should be a full, first class DicomAccess that can both read and write the file system, so I think you will have to make the additional methods public.

Reply:
I mean public on the DicomAccess, but unimplemented by default.